src/Task1/SelectGenre.jsx

```diff
@@ -1,24 +1,29 @@
 import React, { useState } from "react";
 import "./selectGenre.css";
+import checkmark from "./../images/checkmark.svg";
 const SelectGenre = () => {
-  const [show, setShow] = useState("");
-  const [clip, setClippath] = useState("polygon(50% 75%, 0 25%, 100% 25%)");
+  const [isopen, setOpen] = useState(false);
```

Comment:
Please, apply all general JS code formatting and code style guidelines. For example, you should've used `camelCase` of `isopen`. Maybe this one is just a typo. Also, please don't forget about space formatting rules too, like having one empty line after the imports, and visually separating logical blocks of code with an empty line between them.